sys-process/runit/runit-2.2.0.ebuild

```diff
@@ -50,22 +48,39 @@ src_install() {
 	dobin $(<../package/commands)
 	dodir /sbin
 	mv "${ED}"/usr/bin/{runit-init,runit,utmpset} "${ED}"/sbin/ || die "dosbin"
-	if use split-usr ; then
-		dosym ../etc/runit/2 /sbin/runsvdir-start
+	if use scripts ; then
+		if use split-usr ; then
+			dosym ../etc/runit/2 /sbin/runsvdir-start
+		else
+			dosym ../../etc/runit/2 /sbin/runsvdir-start
+		fi
 	else
-		dosym ../../etc/runit/2 /sbin/runsvdir-start
+		exeinto /sbin
+		newexe "${FILESDIR}"/2-${ver_runit_cfg} 2
 	fi
 
 	DOCS=( ../package/{CHANGES,README,THANKS} )
 	HTML_DOCS=( ../doc/*.html )
 	einstalldocs
 	doman ../man/*.[18]
 
-	exeinto /etc/runit
-	doexe "${FILESDIR}"/ctrlaltdel
-	newexe "${FILESDIR}"/1-${ver_runit_cfg} 1
-	newexe "${FILESDIR}"/2-${ver_runit_cfg} 2
-	newexe "${FILESDIR}"/3-${ver_runit_cfg} 3
+	if use scripts ; then
+		exeinto /etc/runit
+		doexe "${FILESDIR}"/ctrlaltdel
+		newexe "${FILESDIR}"/1-${ver_runit_cfg} 1
+		newexe "${FILESDIR}"/2-${ver_runit_cfg} 2
+		newexe "${FILESDIR}"/3-${ver_runit_cfg} 3
+		doexe "${FILESDIR}"/rc.sh
+		insinto /etc/runit/rc
+		doins "${FILESDIR}"/1.openrc.example
+		doins "${FILESDIR}"/3.openrc.example
+
+		# make sv command work
+		newenvd - 20runit <<- EOF
```

Comment:
I think this should be installed always (i.e. it should not depends on USE=scripts).

Reply:
> I think this should be installed always (i.e. it should not depends on USE=scripts).

if USE="-script", SVDIR is determined by the user's script, so I remove it.

Reply:
The `/etc/env.d/20runit` can be easily edited by the user or user can overwrite this env var in `/etc/env.d/99local`, so installing this file is harmless and won't prevent any customizations.

My point is, runit has two major features: boot scripts and service management. USE=scripts should control only boot scripts (`/etc/runit/*`). So it's okay to install both `/etc/env.d/20runit` and even `/sbin/runsvdir-start` with USE=-scripts.